src/containers/BooksForm.js

```diff
@@ -1,19 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { CREATE_BOOK } from '../actions';
 
 const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
 
-const BooksForm = () => (
+const BooksForm = () => {
+  const dispatch = useDispatch();
+
+  const [bookName, setBookName] = useState('');
+  const [bookCat, setBookCat] = useState('');
```

Comment:
- [ ] [OPTIONAL] Good code is legible code, and the same goes to variables.
  - Consider naming your bookCat state to something more descriptive such as bookCategory or simply category.